frontend/src/components/CalcSpectrum.tsx

```diff
@@ -187,6 +236,20 @@ export const CalcSpectrum: React.FC = () => {
     }
   }, [databaseWatch, modeWatch]);
 
+  const DownloadSpectrum: React.FC = () => (
+    <Button
+      id="down-spectrum-button"
+      disabled={downloadButton}
+      variant="contained"
+      color="primary"
+      onClick={handleSubmit((data) => {
+        console.table(data);
+        onSubmit(data, `download-spectrum`);
+      })}
+    >
+      Download spectrum
```

Comment:
```suggestion
      Download
```
per erwan's comment: https://radis-radiation.slack.com/archives/C03LF0J1C3E/p1660831775354799?thread_ts=1660831242.685479&cid=C03LF0J1C3E